docs/integrations/pytorch_lightning.rst

```diff
@@ -1,318 +1,305 @@
-Neptune-PyTorch Lightning Integration
-=====================================
+PyTorch Lightning Integration
+=============================
 
-The integration enables you to log |PyTorch Lightning| metrics to Neptune.
+What will you get?
+------------------
+[video-placeholder]
 
-.. image:: ../_static/images/integrations/pytorchlightning_neptuneml.png
-   :target: ../_static/images/integrations/pytorchlightning_neptuneml.png
-   :alt: PyTorchLightning neptune.ai integration
+PyTorch Lightning is a lightweight PyTorch wrapper for high-performance AI research. With Neptune integration you can:
 
-Requirements
-------------
-This integration is enabled as part of PyTorch Ligthning loggers module, so all you need is to have |neptune-client| installed.
+* preview running experiment
+* log training and testing metrics, and visualize them in Neptune UI,
+* log experiment parameters,
+* monitor hardware usage,
+* log any additional metrics of your choice,
+* log performance charts,
+* save model checkpoints.
 
-.. code-block:: bash
+.. tip::
 
-    pip install neptune-client
+    Check this public project with example experiments: |project|.
 
+    You can also go straight to the |exp|.
 
-Create the **LightningModule**
-------------------------------
+Quickstart
+----------
 
-Nothing changes here, just your typical ``LightningModule`` structure.
+Introduction
+^^^^^^^^^^^^
+This quickstart will show you how to use Neptune with your existing PyTorch Lightning code. You have three options:
```

Comment:
I think this is not true. 
The quickstart doesn't show you how to connect -> it shows you how to log some experiment to lightning. 

We could have how-to connect as well -> perhaps as a third piece of content but for now I'd rather not have it :)

Reply:
thanks, corrected.